src/components/SignInButtons/AppleSignIn/index.website.js

```diff
@@ -0,0 +1,130 @@
+import React, {useEffect, useState} from 'react';
+import {useIsFocused} from '@react-navigation/native';
+import PropTypes from 'prop-types';
+import Config from 'react-native-config';
+import get from 'lodash/get';
+import getUserLanguage from '../GetUserLanguage';
+import * as Session from '../../../libs/actions/Session';
+import Log from '../../../libs/Log';
+import * as Environment from '../../../libs/Environment/Environment';
+
+// TODO: copied from CONFIG.js, refactor
+// react-native-config doesn't trim whitespace on iOS for some reason so we
+// add a trim() call to lodashGet here to prevent headaches
+const lodashGet = (config, key, defaultValue) => get(config, key, defaultValue).trim();
+
+const requiredPropTypes = {
+    isDesktopFlow: PropTypes.bool.isRequired,
+};
+
+const propTypes = {
+    isDesktopFlow: PropTypes.bool,
+};
+const defaultProps = {
+    isDesktopFlow: false,
+};
+
+// TODO: move to appropriate consts file
```

Comment:
TODO, as stated. Wasn't sure about moving the `lodashGet` to const files though, because the token override isn't overriding a const.